lib/runtime/wazero/instance_test.go

```diff
@@ -1617,3 +1633,16 @@ func getParachainHostTrie(t *testing.T) *trie.Trie {
 
 	return tt
 }
+
+func getParachainHostTrie2(t *testing.T) *trie.Trie {
```

Comment:
I would suggest to change the function `getParachainHostTrie` to receive another param called `storage []Storage`

```go
func getParachainHostTrie(t *testing.T, testDataStorage []Storage) *trie.Trie {
	tt := trie.NewEmptyTrie()
	for _, s := range testDataStorage {
		key := common.MustHexToBytes(s.Key)
		value := common.MustHexToBytes(s.Value)
		err := tt.Put(key, value)
		require.NoError(t, err)
	}

	return tt
}

getParachainHostTrie(t, parachainTestData.Storage)
getParachainHostTrie(t, parachain2TestData.Storage)
```